resources/lang/id/model_validation.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'not_negative' => ':attribute tidak bisa negatif.',
+    'required' => ':attribute diwajibkan.',
+    'too_long' => ':attribute melebihi batas maksimum - hanya bisa hingga :limit karakter.',
+    'wrong_confirmation' => 'Konfirmasi tidak cocok.',
+
+    'beatmap_discussion_post' => [
+        'discussion_locked' => 'Diskusi terkunci',
+        'first_post' => 'Tidak dapat menghapus postingan awal.',
+    ],
+
+    'beatmapset_discussion' => [
+        'beatmap_missing' => 'Keterangan waktu telah ditentukan tetapi beatmap tidak ada.',
+        'beatmapset_no_hype' => 'Beatmap tidak dapat dihype.',
+        'hype_requires_null_beatmap' => 'Hype harus dilakukan di bagian General (All difficulties).',
+        'invalid_beatmap_id' => 'Tingkat kesulitan yang tidak valid ditentukan.',
+        'invalid_beatmapset_id' => 'Beatmap yang tidak valid ditentukan.',
+        'locked' => 'Diskusi dikunci',
+        'mapper_note_wrong_user' => 'Hanya pemilik beatmap yang dapat mengirim catatan.',
+
+        'hype' => [
+            'guest' => 'Silakan masuk untuk dapat memberikan hype.',
+            'hyped' => 'Anda telah memberikan hype untuk beatmap ini.',
+            'limit_exceeded' => 'Anda telah menggunakan semua hype Anda.',
+            'not_hypeable' => 'Beatmap ini tidak dapat dihype.',
+            'owner' => 'Tidak terdapat hype di beatmap Anda.',
+        ],
+
+        'timestamp' => [
+            'exceeds_beatmapset_length' => 'Keterangan waktu yang ditentukan melebihi panjang beatmap.',
+            'negative' => 'Keterangan waktu tidak bisa bernilai negatif.',
+        ],
+    ],
+
+    'forum' => [
+        'feature_vote' => [
+            'not_feature_topic' => 'Hanya dapat memilih permintaan fitur.',
+            'not_enough_feature_votes' => 'Suara tidak cukup.',
+        ],
+
+        'poll_vote' => [
+            'invalid' => 'Opsi tidak valid ditentukan.',
```

Comment:
Opsi yang ditentukan tidak valid.

Reply:
Should i change the whole "opsi" to "pilihan" too, or just this one?